app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const serverInfo = require("./db/server.js")
```

Comment:
Do we need to bring this listener into the app? Supertest will attach it's own listener 💭

Reply:
I was struggling to get this to work in any other way than this. This is annoyingly the main culprit as to why test 2 is getting redone so late. 😵